### ISSUE TITLE
TypeError: must be real number, not position

### ISSUE THREAD

**Issue by bober-gith** [renpy/renpy#5306]
```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/main/script.rpy", line 272, in script
    $renpy.pause(delay = 0.5, hard = True)
  File "game/script/main/script.rpy", line 272, in <module>
    $renpy.pause(delay = 0.5, hard = True)
TypeError: must be real number, not position

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/main/script.rpy", line 272, in script
    $renpy.pause(delay = 0.5, hard = True)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script/main/script.rpy", line 272, in <module>
    $renpy.pause(delay = 0.5, hard = True)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\exports.py", line 1661, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\core.py", line 2814, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\core.py", line 1352, in draw_screen
    surftree = renpy.display.render.render_screen(
  File "render.pyx", line 486, in renpy.display.render.render_screen
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\layout.py", line 900, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\layout.py", line 900, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\layout.py", line 900, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\screen.py", line 754, in render
    child = renpy.display.render.render(self.child, w, h, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\layout.py", line 900, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\layout.py", line 1121, in render
    surf = render(d, width - x, rh, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\behavior.py", line 2310, in render
    thumb = render(self.style.thumb, width, thumb_dim, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\image.py", line 513, in render
    return wrap_render(self.target, width, height, st, at)
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\image.py", line 316, in wrap_render
    rend = render(child, w, h, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 260, in renpy.display.render.render
  File "C:\Users\alcia\Downloads\renpy-8.2.0-sdk\renpy\display\transform.py", line 948, in render
    return RenderTransform(self).render(width, height, st, at)
  File "accelerator.pyx", line 966, in renpy.display.accelerator.RenderTransform.render
  File "accelerator.pyx", line 401, in renpy.display.accelerator.RenderTransform.render_child
  File "render.pyx", line 219, in renpy.display.render.render
TypeError: must be real number, not position

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012501
Thu Jan 25 12:55:12 2024
```

When setting define config.mixed_position = False
It solves it, still it seems a bit odd that this would cause a problem.

**Comment by mal** [comment]
Is it possible that you have `config.relative_transform_size` set to `False` in your game?

**Comment by mal** [comment]
Thanks for confirming. I suspect the issue stems from this section of code, where the conversion is being skipped if that `config` variable is set to `False`,
https://github.com/renpy/renpy/blob/cf6312c40d11bb85936d46cd2f9faf6b7c8fa0ab/renpy/display/accelerator.pyx#L391-L399

**Comment by renpytom** [comment]
I believed this is fixed, and I updated the prerelease to 8.2.0.24012502.
